lib/dialects/postgres/query-generator.js

```diff
@@ -604,15 +604,15 @@ class PostgresQueryGenerator extends AbstractQueryGenerator {
     return `ALTER TRIGGER ${this.quoteIdentifier(oldTriggerName)} ON ${this.quoteTable(tableName)} RENAME TO ${this.quoteIdentifier(newTriggerName)};`;
   }
 
-  createFunction(functionName, params, returnType, language, body, optionsArray, options) {
+  createFunction(functionName, params, variables, returnType, language, body, optionsArray, options) {
```

Comment:
This addition of a new parameter in the middle of others is not a good idea... Even if the ordering might feel logic, this would be a breaking change, because everyone would have to change their calls to fit the new argument list. So instead, I suggest you make `variables` a new option under the `options` parameter (the last one). This way your change wouldn't be breaking, and will be a standard feature PR.